### PR TITLE
Merge IDL CSS classes

### LINKED ISSUE #1892: Merge equivalent CSS classes for IDL
Currently the IDL text contains several different CSS classes that does the same, for example:

```css
.idlInterfaceID,
.idlDictionaryID,
.idlCallbackID,
.idlEnumID {
  font-weight: bold;
  color: #005a9c;
}

/*.idlIterable*/

.idlIterableKeyType,
.idlIterableValueType {
  color: #005a9c;
}

/*.idlMaplike*/

.idlMaplikeKeyType,
.idlMaplikeValueType {
  color: #005a9c;
}

.idlAttrType,
.idlFieldType,
.idlMemberType {
  color: #005a9c;
}

.idlConstType {
  color: #005a9c;
}

.idlTypedefID,
.idlTypedefType {
  color: #005a9c;
}

.idlAttrName,
.idlFieldName,
.idlMemberName {
  color: #ff4500;
}

.idlAttrName a,
.idlFieldName a,
.idlMemberName a {
  color: #ff4500;
  border-bottom: 1px dotted #ff4500;
  text-decoration: none;
}
```

Recently I has been trying #1702 and found this complicates the code by forcing different class names for each IDL type names.

Maybe we can do something like:

```css
.idlID {
  font-weight: bold;
  color: #005a9c;
}

.idlContainer .idlID {
  font-weight: normal;
}
```

... so that we can use same CSS classes for IDL type names?

### PR DESCRIPTION
**Pull request by saschanaz**
Fixes #1892

### CODE REVIEW

**Comment by saschanaz** [comment]
Looks like eslint behaves differently in Travis? Codacy and `npm run lint` in my local environment caught the issue and Travis didn't. 🤔

File: js/core/templates/webidl-contiguous/method.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <span class='idlMethod' id="{{obj.idlId}}" data-idl data-title='{{name}}'>{{extAttr obj
-}}{{{special}}}{{#if obj.body}}<span class='idlMethType'>{{idlType obj.body}}</span> {{#if name
-}}<span class='idlMethName'>{{#tryLink obj}}{{name}}{{/tryLink}}</span>{{/if}}({{{children}}}){{/if}};</span>
\ No newline at end of file
+}}{{{special}}}{{#if obj.body}}<span class='idlType'>{{idlType obj.body}}</span> {{#if name
+}}<span class='idlName'>{{#tryLink obj}}{{name}}{{/tryLink}}</span>{{/if}}({{{children}}}){{/if}};</span>
```
Comment (marcoscaceres) [inline js/core/templates/webidl-contiguous/method.html]: Careful with the line endings.

Comment (marcoscaceres) [reply js/core/templates/webidl-contiguous/method.html]: (might be ok... but they've given us sadness before)

Comment (saschanaz) [reply js/core/templates/webidl-contiguous/method.html]: I had to turn off insertFinalNewLine option on VSCode to keep the line endings intact 😅

I hope we get #1702 fixed soon to get rid of them!

**Pull request merged**